analysis/README.md

```diff
@@ -99,6 +99,7 @@ ASM to PIL has two steps: ROM generation and reduction to constrained.
 ##### ROM generation
 
 Rom generation generates a single ROM for each virtual machine using the following process:
+
```

Comment:
Same with all the newline changes here - I assume they were by accident.

Reply:
Yes, probably the autoformatting when saving did its thing. I'll fix it ASAP